ruby/introduction/installing_ruby.md

```diff
@@ -255,6 +255,14 @@ echo 'eval "$(rbenv init -)"' >> ~/.zshrc
 
 You'll notice nothing happened in the terminal. That's okay and is typical response for many terminal commands. At this point, take note of the page and step number you are on, close everything, do a full reboot and log back into your profile. After logging back in, re-open the terminal (see Step 1.1).
 
+Another option is that you see this message:
+
+```bash
+writing ~/.zprofile: now configured for rbenv.
+```
+
+In that case, you're all set and can continue to the next step.
+

```

Comment:
I'm fine with keeping your change above about how Foundations already has learners install homebrew and xcode, but I'm gonna request that you remove these lines. 

Rbenv has changed its `init` script and setup, and our instructions should change to match what they've done. This will be something that's better addressed through #28436  -- After we make the changes required by that issue, this change you've proposed here will no longer make sense.

Let me know if you have any questions/concerns

Reply:
That makes sense, they have been removed!